src/ios/GCDWebServerFramework/GCDWebServer.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,484 @@
+			projectDirPath = "";
+			projectRoot = "";
+			targets = (
+				87CE76BC1C282FDC00FA34FD /* GCDWebServer */,
+			);
+		};
+/* End PBXProject section */
+
+/* Begin PBXResourcesBuildPhase section */
+		87CE76BB1C282FDC00FA34FD /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				87F501AA1C295DA600779832 /* build_configuration.xcconfig in Resources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXResourcesBuildPhase section */
+
+/* Begin PBXShellScriptBuildPhase section */
+		87CE76C81C28306800FA34FD /* Create Universal Framework  */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 12;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "Create Universal Framework ";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "#Don't execute script if CREATE_UNIVERSAL_FRAMEWORK is 0 in order to avoid infinite loops.\nif [ \"${CREATE_UNIVERSAL_FRAMEWORK}\" = 0 ]; then\nexit $?\nfi\n\n#For some reason xcodebuild fails to produce simulator build (i386,x86_64) when one chooses 'Generic iOS device' from Xcode 7.1\n#Xcode error:\n#CodeSign error: entitlements are required for product type 'Framework' in SDK 'Simulator - iOS 9.1'. Your Xcode installation may be damaged.\n\nif [[ \"${SDK_NAME}\" == iphoneos* ]]; then\necho \"Build for simulator in order to produce universal static framework.\"\nexit 1;\nfi\n\nif [[ \"${CONFIGURATION}\" != Release ]]; then\necho \"Change build configuration to 'Release'.\"\necho \"Go to Edit Scheme...->Run->Build Configuration\"\nexit 1;\nfi\n\n\n#Constants.\nFRAMEWORK_NAME=\"${PROJECT_NAME}.framework\"\nSIMULATOR_FRAMEWORK_PATH=\"${BUILD_DIR}/${CONFIGURATION}-iphonesimulator\"\nDEVICE_FRAMEWORK_PATH=\"${BUILD_DIR}/${CONFIGURATION}-iphoneos\"\nUNIVERSAL_FRAMEWORK_PATH=\"$(dirname \"${SRCROOT}\")\"\n\n#Produce builds for all simulator and arm architectures.\nxcodebuild -project ${PROJECT_NAME}.xcodeproj -sdk iphonesimulator -arch i386 -arch x86_64 -target ${PROJECT_NAME} -configuration ${CONFIGURATION}  CONFIGURATION_BUILD_DIR=${SIMULATOR_FRAMEWORK_PATH} CREATE_UNIVERSAL_FRAMEWORK=0 clean build\n\nxcodebuild -project ${PROJECT_NAME}.xcodeproj -sdk iphoneos -arch arm64 -arch armv7 -arch armv7s -target ${PROJECT_NAME} -configuration ${CONFIGURATION} CONFIGURATION_BUILD_DIR=${DEVICE_FRAMEWORK_PATH} CREATE_UNIVERSAL_FRAMEWORK=0 clean build\n\n#Clean destination directory.\nrm -rf \"${UNIVERSAL_FRAMEWORK_PATH}/${FRAMEWORK_NAME}\" && mkdir -p \"${UNIVERSAL_FRAMEWORK_PATH}/${FRAMEWORK_NAME}\"\ncp -r \"${DEVICE_FRAMEWORK_PATH}/${FRAMEWORK_NAME}/.\" \"${UNIVERSAL_FRAMEWORK_PATH}/${FRAMEWORK_NAME}\"\n\n#Produce universal binary.\nlipo \"${SIMULATOR_FRAMEWORK_PATH}/${FRAMEWORK_NAME}/${PROJECT_NAME}\" \"${DEVICE_FRAMEWORK_PATH}/${FRAMEWORK_NAME}/${PROJECT_NAME}\" -create -output \"${UNIVERSAL_FRAMEWORK_PATH}/${FRAMEWORK_NAME}/${PROJECT_NAME}\"";
```

Comment:
I recommend that you extract the script in a `.sh` file and call it from here. Otherwise it'll be much more difficult to read and maintain.